tests/test_FindTest.py

```diff
@@ -224,12 +249,96 @@ def test_find_citations(self):
             ('before asdf, 1 U. S., at 20-25',
              [case_citation(2, page='20', reporter_found='U. S.', short=True,
                             metadata={'pin_cite': '20-25',
-                                      'antecedent_guess': 'asdf'})]),
+                                      'antecedent_guess': 'asdf',
+                                      'court': 'scotus'})]),
             # Test short form citation with a page range with weird suffix
             ('before asdf, 1 U. S., at 20-25\\& n. 4',
              [case_citation(2, page='20', reporter_found='U. S.', short=True,
                             metadata={'pin_cite': '20-25',
-                                      'antecedent_guess': 'asdf'})]),
+                                      'antecedent_guess': 'asdf',
+                                      'court': 'scotus'})]),
+            # Test short form citation with a parenthetical
+            ('before asdf, 1 U. S., at 2 (overruling xyz)',
+             [case_citation(4, page='2', reporter='U.S.',
+                            reporter_found='U. S.', short=True,
+                            metadata={'antecedent_guess': 'asdf',
+                                      'parenthetical': 'overruling xyz',
+                                      'court': 'scotus'}
+                            )]),
+            # Test short form citation with no space before parenthetical
```

Comment:
Looks like this test string should have a space removed?

Reply:
Fixed!